### ISSUE TITLE
The demo page and main.less file should be more dynamic

### ISSUE THREAD

**Issue by himedlooff** [cfpb/generator-cf#34]
For example if you don't choose all of the components you could get an error for missing variables. Also you will still see the demo markup for that component which will be unstyled.

**Comment by Scotchester** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
Ran into a missing variable problem for `@super-btn-font-size` in src/static/css/cf-theme-overrides.less when I didn't select buttons as a component.

**Comment by himedlooff** [comment]
Good catch @KimberlyMunoz. @Scotchester fixed this issue here: https://github.com/cfpb/generator-cf/pull/35

You can make the same change locally. Let me know if this solves it for you.
